common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/protocol/AbstractFetchShuffleBlocks.java

```diff
@@ -43,12 +43,14 @@ protected AbstractFetchShuffleBlocks(
     this.shuffleId = shuffleId;
   }
 
+  // checkstyle.off: RegexpSinglelineJava
   public ToStringBuilder toStringHelper() {
```

Comment:
This should probably not be a public api, since deleting it doesn't cause a mima check failure.

Reply:
Ya, I hope to remove this eventually later because this method is not used from now.

However, this PR aims to focus on the improvement only to avoid any discussion about this method.

https://github.com/apache/spark/blob/c7176246a21aee6b48dfe957f2baf4e547a48e4b/common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/protocol/AbstractFetchShuffleBlocks.java#L27-L46